djangocms_bootstrap4/contrib/bootstrap4_card/blueprints.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
```

Comment:
please ignore this for now, I disabled it in the plugins but want to keep the code for now in the codebase